code/modules/paperwork/photocopier.dm

```diff
@@ -11,7 +11,7 @@
 	power_channel = EQUIP
 	var/obj/item/copyitem = null	//what's in the copier!
 	var/copies = 1	//how many copies to print!
-	var/toner = 30 //how much toner is left! woooooo~
+	var/toner = 60 //how much toner is left! woooooo~
 	var/maxcopies = 10	//how many copies can be copied at once- idea shamelessly stolen from bs12's copier!
 	var/mob/living/ass = null
```

Comment:
since you're using `atom_say`, you may want to change the `atom_say_verb` to something like "beeps", since it's a machine--the default is just "says". This is ultimately up to you, of course.